mozilla-release/browser/app/module.ver

```diff
@@ -1,8 +1,7 @@
-WIN32_MODULE_COMPANYNAME=Mozilla Corporation
-WIN32_MODULE_COPYRIGHT=�Firefox and Mozilla Developers; available under the MPL 2 license.
+WIN32_MODULE_COMPANYNAME=Cliqz GmbH
+WIN32_MODULE_COPYRIGHT=Cliqz GmbH
 WIN32_MODULE_PRODUCTVERSION=@MOZ_APP_WINVERSION@
 WIN32_MODULE_PRODUCTVERSION_STRING=@MOZ_APP_VERSION@
-WIN32_MODULE_TRADEMARKS=Firefox is a Trademark of The Mozilla Foundation.
```

Comment:
This constant is still used in https://github.com/cliqz-oss/browser-f/blob/DB-593_1/mozilla-release/config/version_win.pl#L160